src/curve/constant-product/ConstantProductPair.sol

```diff
@@ -107,7 +107,6 @@ contract ConstantProductPair is ReservoirPair {
                 if (lSqrtNewK > lSqrtOldK) {
                     uint256 lSharesToIssue = _calcFee(lSqrtNewK, lSqrtOldK, platformFee, totalSupply);
 
-                    // TODO: What happens if no PLATFORM_FEE_TO is set
```

Comment:
for this, some platformFee would be minted to the zero address. 

We can just make sure to set a `platformFeeTo` in the factory right from the start so that this wouldn't happen. Then we wouldn't need to bloat the curves' code with a check that it is non-zero

Reply:
Sounds good